tooltester/README.md

```diff
@@ -2,27 +2,60 @@
 ---
 **NOTE:**
 
-This `README.md` only shows users how to use the `run-workflows` command. As of April 3, 2023 all other 
+This `README.md` only shows users how to use the `run-workflows-through-wes` command. As of April 3, 2023 all other 
 commands are no longer working and will require a non-trivial amount of work to get working.
 
 The README for these features can be found here: https://github.com/dockstore/dockstore-support/blob/8775ecadaec8b5fa36d03bc6ba962509ea38ca29/tooltester/README.md
 
 ---
 
-# The Tooltester `run-workflows` Command
+# The Tooltester `run-workflows-through-wes` Command
 
-The purpose of the `run-workflows` command, is to be able to run (in an automated fashion) workflows found on a Dockstore site 
+The purpose of the `run-workflows-through-wes` command, is to be able to run (in an automated fashion) workflows found on a Dockstore site 
 (ie. [dockstore.org](https://dockstore.org/) or [qa.dockstore.org](https://qa.dockstore.org/)), and 
 - Determine what the end state of the workflow (ie. SUCCESS or EXECUTOR_ERROR)
 - Determine how long it takes for a workflow to reach its end state.
 - Gather metrics such as CPU usage of the workflow during its run.
 
-All of the above information is then posted to an `S3` container via the [Execution class](https://github.com/dockstore/dockstore/blob/290bc2c68640e82a44bdb8fe58c8629814739dfa/dockstore-webservice/src/main/java/io/dockstore/webservice/core/metrics/Execution.java#L36-L127).
+All of the above information is then posted to an `S3` container via the [RunExecution class](https://github.com/dockstore/dockstore/blob/a5e7f12b8768ee57f9ce9401d4912e2bd3ec9889/dockstore-webservice/src/main/java/io/dockstore/webservice/core/metrics/RunExecution.java#L37).
 
-## What is Required to run `run-workflows`
+## What is Required to run `run-workflows-through-wes`
+To run the `run-workflows-through-wes` we have to set a few things up first. This can either be done manually or with the `setUpTooltester.sh` script.
+It is **highly recommended** that you use the script.
+
+## How to use the Setup Script
+First, ensure that your `agc-project.yaml` file is in the same directory as `setUpTooltester.sh`.
```

Comment:
If we're going to use your agc-project.yaml, we don't need to ensure anything; I'd say just navigate to the directory with those two files.